packages/drivers/routerlicious-driver/src/urlUtils.ts

```diff
@@ -17,3 +19,23 @@ export const parseFluidUrl = (fluidUrl: string): URLParse => {
  */
 export const replaceDocumentIdInPath = (urlPath: string, documentId: string): string =>
     urlPath.split("/").slice(0, -1).concat([documentId]).join("/");
+
+export const createFluidUrl = (domain: string, pathname: string): string =>
+    "fluid://".concat(domain).concat(pathname);
+
+export const replaceWithDiscoveryUrl = (resolvedUrl: IFluidResolvedUrl,
+    session: ISession,
+    parsedUrl: URLParse): void => {
+    if (session && session.ordererUrl.includes("https")) {
+        const replacementOrderUrl = new URL(session.ordererUrl);
+        const replaceHistorianUrl = new URL(session.historianUrl);
+        const deltaStorageUrl = new URL(resolvedUrl.endpoints.deltaStorageUrl);
```

Comment:
(educate me) why replacement vs replace?

Reply:
Got it. I changed it to replace. :)